src/features/query-bar/__tests__/Condition-test.js

```diff
@@ -75,32 +101,21 @@ describe('features/query-bar/components/filter/Condition', () => {
                 value: '',
             },
         };
-        const index = 0;
         const condition = initialCondition;
         const dateFieldValue = new Date(2018, 11, 24, 10, 33, 30, 0);
-        const valueType = 'string';
-        const keyType = VALUE_KEY;
-        const displayTextType = 'valueDisplayText';
+        const keyType = VALUE;
 
         test.each`
-            description                                            | fieldValue          | displayText       | value
-            ${'should enter an empty string into the value field'} | ${stringFieldValue} | ${''}             | ${''}
-            ${'should select a date in the date picker'}           | ${dateFieldValue}   | ${dateFieldValue} | ${dateFieldValue}
-        `('$description', ({ fieldValue, displayText, value }) => {
+            description                                            | fieldValue          | value
+            ${'should enter an empty string into the value field'} | ${stringFieldValue} | ${''}
```

Comment:
These should be rephrased as:
* "should invoke onFieldChange with empty string"
* "should invoke onFieldChange with Date"